src/servers/src/postgres/server.rs

```diff
@@ -79,6 +80,13 @@ impl PostgresServer {
                 match tcp_stream {
                     Err(error) => error!("Broken pipe: {}", error), // IoError doesn't impl ErrorExt.
                     Ok(io_stream) => {
+                        match io_stream.peer_addr() {
+                            Ok(addr) => info!("PostgreSQL connection coming from {}", addr),
```

Comment:
change this to `debug` or `trace` level? `info` might be annoying as we will `warn` on the error path.